matrix/common/request/request_cache.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, request_id: str) -> None:
         self._dynamo_handler = DynamoHandler()
         self._cloudwatch_handler = CloudwatchHandler()
 
+        self._creation_date = ""
```

Comment:
Why do we need to instantiate an empty string?

Reply:
Variable isn't recognized as a member unless defined in `__init__`